package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "validator-history-service",
-  "version": "0.4.1",
+  "version": "0.4.3",
```

Comment:
Why is this being bumped two patches?

Reply:
Just to match what's on npmjs (not sure if that's necessary). I did 2 publishes this month.

Reply:
In the future, please bump the version in the repo. You should also add a tag at the commit you published at (this can be done for already-done-publishes).